includes/RestApiClient/RestApiClient.php

```diff
@@ -5,8 +5,15 @@
 namespace Dation\Woocommerce\RestApiClient;
 
 use DateTime;
```

Comment:
The lack of dependency injection is quite apparent in these 2 lines. How hard would it be to add the DI-component from Symfony?

Reply:
Maybe next iteration?

Reply:
Maybe next iteration?